src/index.html

```diff
@@ -3,11 +3,63 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+          content="width=device-width,initial-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Product cards</h1>
+
+  <body class="body">
+
+    <main class="main">
+      <div class="card" data-qa="card">
+
+        <div class="img_top">
+          <div class="img">
+          </div>
+        </div>
+
+        <div class="product">
+          <a href="#" class="name">APPLE A1419 iMac 27" Retina <br> 5K Monoblock (MNED2UA/A)</a>
+        </div>
+
+        <div class="code_top">
+          <div class="code">
+            Product code: 195434
+          </div>
+        </div>
+
+        <div class="stat">
+          <div class="star">
+            <div class="active"></div>
```

Comment:
```suggestion
          <div class="stars stars--4">
            <div class="stars__star"></div>
```
reuse code from stars task